tutorials/gas_compact_multiphysics/gold/solid_mesh_out.csv

```diff
@@ -1,3 +1,3 @@
 time,area_walls,volume_coolant,volume_fuel
 0,0,0,0
-1,0.080483429170927,0.00032169908772759,0.00040536598327801
```

Comment:
Please undo the changes to this file. This test was not covering anything related to the wrong thermal conductivity, this is just a mesh test and these diffs are just floating point noise on your machine.

Reply:
I'll fix this too. Assuming the random noise is below the test threshold, which is why the test passes with this change.